scripts/verify-nupkgs.ps1

```diff
@@ -19,7 +19,7 @@ function Verify-Nuget-Packages($packageDirectory)
                      "Microsoft.TestPlatform.Build" = 19;
                      "Microsoft.TestPlatform.CLI" = 350;
                      "Microsoft.TestPlatform.Extensions.TrxLogger" = 33;
-                     "Microsoft.TestPlatform.ObjectModel" = 91;
+                     "Microsoft.TestPlatform.ObjectModel" = 178;
```

Comment:
What is the impact of this on the relative size of the package? Did it grow to be 2 times the size? What is the absolute size?

Reply:
`16.8.0-preview-20200921-01` is 886 KB, and the unsigned package generated with this is 1142 KB; I guess we need to compare the signed versions.